pytradfri/api/aiocoap_api.py

```diff
@@ -145,7 +145,7 @@ def psk(self, value):
 
         _, res = await self._get_response(msg)
 
-        api_command.result = _process_output(res, parse_json)
+        api_command.result, _ = _process_output(res, parse_json)
```

Comment:
Maybe we can check on the `CHANGED` code here too? Although it seems a bit more common than in an observation.

Reply:
After testing, it seems that we will have to check for the `CHANGED` response here instead of inside an observation. Albeit the `CHANGED` response in an observation callback will occur, it is rather rare, but on this spot it is quite common.

I will ammend my PR with that change in the near future.